packages/google_maps_flutter/google_maps_flutter/ios/google_maps_flutter.podspec

```diff
@@ -19,8 +19,10 @@ Downloaded by pub (not CocoaPods).
   s.module_map = 'Classes/google_maps_flutter.modulemap'
   s.dependency 'Flutter'
   s.dependency 'GoogleMaps'
+  s.dependency 'Google-Maps-iOS-Utils'
   s.static_framework = true
   s.platform = :ios, '9.0'
+  s.swift_version = '4.0'
```

Comment:
Do we need this?

Reply:
It appears not. I must have added it while trying to get `Google-Maps-iOS-Utils` to work. Removed.

Reply:
Oh the CI doesn't like not having it that's why I added it 🙃

Reply:
What was the error?  What is requiring this?

Reply:
https://github.com/flutter/plugins/runs/7349570630